packages/manager/apps/catalog-revamp/src/hooks/useCatalog.ts

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from 'react';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import {
+  getManagerHubCatalogList,
+  getManagerHubCatalogListQueryKey,
+} from '@/api';
+import {
+  groupItems,
+  filterByEnum,
+  getAvailableCategories,
+  matchCategories,
+  matchUniverses,
+  matchSearchText,
+  filterItems,
+  getUniverses,
+} from '@/utils/utils';
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+export type CatalogData = {
+  catalog: {
+    data: Product[];
+  };
+};
```

Comment:
**nitpick:** This is the structure of 2API response, since we plan to decommission 2API after PIM prod in US. We dont have to stick to this structure. `CatalogData` can be an array of Products.
```suggestion
export type CatalogData =  Product[];
```

Reply:
When will this update happen to 2API ? Because if I change it right now the code is not working anymore.